### PR TITLE
Feat129 landing page

### PR DESCRIPTION
**Pull request by azheng1991**
Created landing page to solve issue #129, comments and suggestions welcome!

### CODE REVIEW

**Comment by JMStudiosJoe** [comment]
@azheng1991 i think there needs to be a formatting check for the check to pass? is that what is failing here @alexjball ?

**Comment by acharliekelly** [comment]
@JMStudiosJoe, the `fix` script kept inexplicably crashing. Maybe related to whatever is causing checks to fail?

File: frontend/shared-components/index.tsx [hunk 1]
```diff
@@ -17,5 +18,6 @@ export {
   PrimaryButton,
   PrimaryInput,
   ResponseTextArea,
-  USAStateInput
+  USAStateInput,
+ 
 }
```
Comment (acharliekelly) [inline frontend/shared-components/index.tsx:23]: Was this commit on purpose?

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```
Comment (acharliekelly) [inline package-lock.json:3]: Oops, you ran a script on the root folder instead of `frontend`

**Review by acharliekelly — CHANGES_REQUESTED** [review]
Only problem is the failing test. I think there are two unintended commits: `frontend/shared-components/index.tsx`, and the root-level `package-lock.json`. I'd start by rolling those back, and then see if it passes.

File: frontend/tests/snapshots/landing.test.tsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import * as React from "react"
+import { LandingPage} from "../../compositions/"
+import renderer from "react-test-renderer"
+
+it("renders Saved Searches page correctly", () => {
```
Comment (alexjball) [inline frontend/tests/snapshots/landing.test.tsx:5]: Change this to landing page

File: frontend/tests/snapshots/landing.test.tsx [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+  const tree = renderer.create(<LandingPage />).toJSON()
```
Comment (alexjball) [inline frontend/tests/snapshots/landing.test.tsx:6]: Nit: use the render method from test-utils, it generates a more accurate snapshot. Take a look at profile.test.tsx

**Comment by alexjball** [comment]
You'll need to run npm run fix to fix the formatting. If that is giving you errors, please paste the output in a comment

**Review by alexjball — DISMISSED** [review]

File: frontend/tests/snapshots/landing.test.tsx (hunk 3 above)
Comment (azheng1991) [reply frontend/tests/snapshots/landing.test.tsx:5]: ok changed

File: frontend/tests/snapshots/landing.test.tsx (hunk 4 above)
Comment (azheng1991) [reply frontend/tests/snapshots/landing.test.tsx:6]: ok fixed

**Comment by azheng1991** [comment]
still getting some errors when running npm run fix. Output of the debug log: 
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'fix'
1 verbose cli ]
2 info using npm@6.14.13
3 info using node@v14.17.3
4 verbose run-script [ 'prefix', 'fix', 'postfix' ]
5 info lifecycle pdc@0.1.0~prefix: pdc@0.1.0
6 info lifecycle pdc@0.1.0~fix: pdc@0.1.0
7 verbose lifecycle pdc@0.1.0~fix: unsafe-perm in lifecycle true
8 verbose lifecycle pdc@0.1.0~fix: PATH: C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;C:\police-data-trust\frontend\node_modules\.bin;C:\Python39\Scripts\;C:\Python39\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Git\cmd;C:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Users\azhen\AppData\Local\Microsoft\WindowsApps;C:\Users\azhen\AppData\Roaming\npm;C:\Users\azhen\AppData\Local\Programs\Microsoft VS Code\bin
9 verbose lifecycle pdc@0.1.0~fix: CWD: C:\police-data-trust\frontend
10 silly lifecycle pdc@0.1.0~fix: Args: [ '/d /s /c', 'next lint --fix; prettier --write .' ]
11 silly lifecycle pdc@0.1.0~fix: Returned: code: 1  signal: null
12 info lifecycle pdc@0.1.0~fix: Failed to exec fix script
13 verbose stack Error: pdc@0.1.0 fix: `next lint --fix; prettier --write .`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:375:28)
13 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:375:28)
13 verbose stack     at maybeClose (internal/child_process.js:1055:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
14 verbose pkgid pdc@0.1.0
15 verbose cwd C:\police-data-trust\frontend
16 verbose Windows_NT 10.0.19042
17 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "fix"
18 verbose node v14.17.3
19 verbose npm  v6.14.13
20 error code ELIFECYCLE
21 error errno 1
22 error pdc@0.1.0 fix: `next lint --fix; prettier --write .`
22 error Exit status 1
23 error Failed at the pdc@0.1.0 fix script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]


Error in terminal: 
> pdc@0.1.0 fix C:\police-data-trust\frontend
> next lint --fix; prettier --write .

Unknown or unexpected option: --fix;
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! pdc@0.1.0 fix: `next lint --fix; prettier --write .`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the pdc@0.1.0 fix script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\azhen\AppData\Roaming\npm-cache\_logs\2021-10-01T19_33_55_457Z-debug.log

**Review by acharliekelly — APPROVED** [review]

**Pull request merged**